docs/wikidata.md

```diff
@@ -26,27 +26,23 @@ build the index under a different path.
     cd qlever
     docker build -t qlever .
 
-## Download and uncompress Wikidata
+## Download and Wikidata
 
-If you already downloaded **and decrompressed** Wikidata to the uncompressed
-Turtle format, you can skip this step. Otherwise we will download and uncompress
-it in this step.
+If you already downloaded Wikidata in the bzip2-compressed
+Turtle format, you can skip this step. Otherwise we will download in this step.
 
 **Note:** This takes several hours as Wikidata is about 42 GB compressed and
-their servers are throttled.
-
-**This is the first step that needs significant amounts of storage.**
-Together, the unpacked Wikidata Turtle file, created in this step, and the index
-we will create in the next section, will use up to about 2 TB.
+their servers are throttled. In case you are in a hurry you can directly pipe the file
+into the index-building pipeline (see next section). But in this case you'll have to redownload
```

Comment:
This sounds too negative I think. It shouldn't go wrong after all. Maybe just remove that last sentence. We could say that it's always a good idea to keep the original input for comparisons and reproducibility, but that sounds almost condescending.